### ISSUE TITLE
[4.0] Latest nightly build fails for TYPO3 fatal error

### ISSUE THREAD

**Issue by joeforjoomla** [joomla/joomla-cms#25248]
### Steps to reproduce the issue
Install the latest nightly build

### Expected result

Just works as previous ones

### Actual result

Fatal error
![image](https://user-images.githubusercontent.com/17835460/59586398-2ab52000-90e3-11e9-94d8-2ce1d5315f1e.png)

### System information (as much as possible)

This code in the bootstrap.php and cms.php causes the error, the TYPO3 package is not included at all

```
// Suppress phar stream wrapper for non .phar files
$behavior = new \TYPO3\PharStreamWrapper\Behavior;
\TYPO3\PharStreamWrapper\Manager::initialize(
	$behavior->withAssertion(new \TYPO3\PharStreamWrapper\Interceptor\PharExtensionInterceptor)
);
if (in_array('phar', stream_get_wrappers()))
{
	stream_wrapper_unregister('phar');
	stream_wrapper_register('phar', 'TYPO3\\PharStreamWrapper\\PharStreamWrapper');
}
```

**Comment by infograf768** [comment]
Did not test Nightly build, but got the same error after the 3.9.3 and 3.9.4 merges into 4.0 in my localhost instance clean install.
I solved it by using composer update and accepting to discard changes as I got this in Terminal
```
The package has modified files: 
M src/drone-run.sh 
Discard changes [y,n,v,d,?]? 
```

**Comment by brianteeman** [comment]
do a composer update - that fixed it for me

**Comment by infograf768** [comment]
Tested Nightly build. It should not happen with a Nightly build as one does not have to use composer in this case.

**Comment by joeforjoomla** [comment]
Indeed. Just now i have not composer to use.

**Comment by infograf768** [comment]
Informed Maintainers of this issue.

**Comment by joeforjoomla** [comment]
Thanks JM

**Comment by infograf768** [comment]
@franz-wohlkoenig 
Why closing? This is a real issue. Re-opening.

**Comment by ghost** [comment]
thought its a Composer-issue.

**Comment by infograf768** [comment]
It is an issue with building Nightly. As it prevents from even installing J, I think it is better to leave it open until @wilsonge solves it, so that testers using only nightlies know about the issue.

**Comment by joeforjoomla** [comment]
Running composer update fixes the issue. The missing Typo3 library in the nightly build is correctly installed.

**Comment by wilsonge** [comment]
This is my bad. In the conflict fixing somehow the typo 3 package ended up in the dev dependencies section (which doesn’t ship in the production & nightly builds). I probably need to regenerate the lock file. But I’m not going to be able to today as I’m seeing family and heading back home in London. If anyone wants to pick it up. Just delete the lock file and run composer install and commit the new one. We’ll probably update some packages on the way but we’re still in active development so it’s no big deal

**Comment by HLeithner** [comment]
I updated the phar wrapper to 3.1.2 that should fix it, correct @wilsonge ?

**Comment by mbabker** [comment]
Nightlies are being rebuilt with https://github.com/joomla/joomla-cms/pull/25251 merged right now, updated package will be on the server within 2-3 minutes of this post.  Note https://github.com/joomla/joomla-cms/pull/25252 is included which bumps PHP minimum requirement to 7.2.

Closing.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25248">issues.joomla.org/tracker/joomla-cms/25248</a>.</sub>
